third_party/move/move-vm/runtime/src/native_functions.rs

```diff
@@ -132,7 +132,12 @@ impl<'a, 'b, 'c> NativeContext<'a, 'b, 'c> {
     ) -> VMResult<(bool, Option<NumBytes>)> {
         let (value, num_bytes) = self
             .data_store
-            .load_resource(self.resolver.loader(), address, type_)
+            .load_resource(
+                self.resolver.loader(),
+                address,
+                type_,
```

Comment:
why _ in the end?

Reply:
I believe `type` is a reserved keyword in rust and you can't use that as variable name.

Reply:
curious to why whoever originally wrote this didn't just use a different variable name :) imho x_, _x, etc, are usually more likely to clash with some coding standards.